chia/util/struct_stream.py

```diff
@@ -13,33 +12,67 @@ def __trunc__(self) -> int:
         ...
 
 
+packing_strings: Dict[Tuple[int, bool], str] = {
+    (1, False): "!B",
```

Comment:
Do we need to use PACK? It seems very confusing to me. What's wrong with `int.to_bytes`?